app/graphql/types/mutation_type.rb

```diff
@@ -2,11 +2,11 @@
 
 module Types
   class MutationType < Types::BaseObject
-    # TODO: remove me
-    field :test_field, String, null: false,
-                               description: 'An example field added by the generator'
-    def test_field
-      'Hello World'
-    end
+    # # TODO: remove me
```

Comment:
I commented out this code because I was not using mutations yet and did not want to somehow throw myself unexpected errors. This may have been unnecessary but I felt more confident continuing with this setup.